source/assets/javascripts/main.js

```diff
@@ -0,0 +1,54 @@
+class ContentNavigation {
+  constructor(element) {
+    this.fixedClass = "fixed";
+    this.navigationBreakPoint = 48.8;
+    this.$navigation = $(element);
+    this.$triggers = this.$navigation.find("a");
+
+    this._bindEvents();
+  }
+
+  get pastNavigationPoint() {
+    return $(window).scrollTop() > this.navigationBreakPoint;
+  }
+
+  _bindEvents() {
+    this.$triggers.on("click", this._animate);
+
+    $(document).on(
+      "scroll ready", _.throttle(this._setNavigationClass.bind(this), 100)
```

Comment:
From a quick search of the repo, I didn't see `lodash` loaded anywhere. A `_.*` is typically a sign that the `lodash` library is being used. Here it looks like we are using https://lodash.com/docs#throttle. 

Is `lodash` in the app?